src/gflownet/tasks/config.py

```diff
@@ -72,16 +48,13 @@ class SEHMOOTaskConfig:
     max_train_it: Optional[int] = None
     n_valid: int = 15
     n_valid_repeats: int = 128
-    objectives: List[str] = field(default_factory=lambda: ["seh", "qed", "sa", "wt"])
+    objectives: List[str] = field(default_factory=lambda: ["seh", "qed", "sa", "mw"])
```

Comment:
Just curious why do you need a factory and lambda versus simply: ` objectives: List[str] = ["seh", "qed", "sa", "mw"]`?

Reply:
^ unless it has changed recently, i believe you cannot have mutable variables as default.

Reply:
Yeah correct: https://stackoverflow.com/questions/69564830/python-dataclass-setting-default-list-with-values

Reply:
Oki thanks!